deps/semver.lua

```diff
@@ -29,9 +30,9 @@ exports.normalize = normalize
 function exports.gte(first, second)
   if not second or first == second then return true end
   if not first then return false end
-  local a, b, c = parse(second)
-  local d, e, f = parse(first)
-  return (d > a) or (d == a and (e > b or (e == b and f > c)))
+  local a, b, c, x = parse(second)
+  local d, e, f, y = parse(first)
+  return (d > a) or (d == a and (e > b or (e == b and (f > c or (y > x)))))
```

Comment:
I think this needs more, what about when `f == c` and `f < c`.  Shouldn't those be treated differently?